docs/deployment_guide/partner_editable/troubleshooting.adoc

```diff
@@ -96,18 +96,24 @@ Verify that an RSA key pair used during deployment. If not, try the following op
 // git log --reverse origin/main...v1.4.0 -- ':!/ci/'
 
 // Changelog links
+:imds-v2: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html
+:tap-1_4_2-cl: https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/release-notes.html#v142-0
 :tap-1_4_1-cl: https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/release-notes.html#v141-0
 :ce-1_4_1-cl: https://docs.vmware.com/en/Cluster-Essentials-for-VMware-Tanzu/1.4/cluster-essentials/release-notes.html#v141-0
 :eks-1_24-cl: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-1.24
 :eksqs-5_0_0-cl: https://github.com/aws-quickstart/quickstart-amazon-eks/releases/tag/v5.0.0
 
+* Version 1.4.2
+** Version upgrades to:
+*** {tap-1_4_2-cl}[{partner-product-acronym} 1.4.2^] 
+** Support for {imds-v2}[IMDSv2^]
 * Version 1.4.1
 ** The stack reports back if the TAP deployment and the installation of the
    sample application succeeded after all the infrastructure has been deployed
 ** Deletion issues, where VPCs could not be cleaned up because of left-over
    security groups, has been fixed
 ** Version upgrades to:
-*** {tap-1_4_1-cl}[{partner-product-acronym}^] 1.4.1
+*** {tap-1_4_1-cl}[{partner-product-acronym} 1.4.1^] 
```

Comment:
*nit*/*non-blocking*: The other hyperlinks below (to CE, EKS QS, EKS) only the name is a link, the version isn't. Here, everything (name + version) is a link. I would prefer if we'd stay consistent.